xsdgen/example_test.go

```diff
@@ -1,12 +1,11 @@
 package xsdgen_test
 
 import (
+	"aqwari.net/xml/xsdgen"
 	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
-
-	"aqwari.net/xml/xsdgen"
```

Comment:
The original order of import was following standard import patters, maybe it should be left as it is

Reply:
Done